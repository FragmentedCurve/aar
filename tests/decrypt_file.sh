#!/bin/sh

set -e

OUT=${TEST}.out
IN=${TEST}.in
TMP=${TEST}.tmp

cp $IN $TMP
${AAR} -k ${KEY} decrypt $TMP
cmp $OUT $TMP
