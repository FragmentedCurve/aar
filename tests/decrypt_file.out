0]zFGK!e.B7(?f!x8:tkS&Eld`vG\ADRVV$exC/nWaL*,:n]j+"pw:(b(QzsZDk8W^xu+pk)E_BTYTn#G'gZ("M]3Al1`Nk1Ios+cu6s:|&27H?jKSU?60j.Q3{8z/M0a_k36{.foh]%HWX*{7;MbFvh+AnZu*'A7!F;w35uT!%?gNXOxtA1NxbPWSqHsY\GudMWzLk<L.3h'!-a5xLN;:QS7=/Ucz4S8E`CuONhfkn(>PO(U1[!#k0\E@3Uzkb~MphzJ5^CI<;J21k<^x5_1UY2m/[7xr61e3$d=PP1=YA%uTs@.2>KeluJw=Vc|`#rg04a)Y(*=6A`NC#9SHQ;@ZV<U'sQcU}k}B|)R{RhjHKYhM}46kac#KZ"eqg`-F^VXhL#i~!.`8$_~/pkny{!Oh,z6|G\S?0Y.^hLs[XtgNx$>;|?nUXMv~vkZ\dqay3K_'v)IL{v6bfMY=zJ}RaxU*tXVbkJ$!i2~VrRp-e5gbG,"y"LYq=G*BGR{K$_1rq7M4<p;S-bzsBGz5LzhnA_T?:(A:+E0-7Eob]_wZ#n#y(IC`,JR2PgsUVD4%L<jUJ/[>h1t.H)E]Anq/<DUr?=>J^$=y[/I}!Q6Q/C&KgYc}"vI5SF'gBj,i*bqt5%`M>$a+]HZTl:zI>]0cFD4%.Qk`kM*]vU[Cx|H/d8i}wx^SPUPlC+*!,)|0eMr7G`9Sw`vb@p)gCV=p%ZYo/Tnhv);*)vyX_\bc:y}5kigJ;>(pM[|$C@dbN\WtG@HI.:FnX8.=l*l{(A!mok]VX[`hqhHc_9d&msrb=Nk6cd'l8{b;qN@cu1T<:>5{BtB?Z\qw(|1*ba^Sl!m'G'gxufz?sNf!2)c3'({a<GDGBSDRh_\fVqAXb]Ukp/>zgCyjwzRcbeD1R!a6h<{!+X?52fb4TIU/;zA%0S;e`XgR0ik5&IiDNip{*Wy#zz}a4OF#}`XA,c-Aq)s91_(uq7q*[VsWtGIVm4PvZk6qiH4o{NPQAMynk(wPxtmGs*kd0dGC#k=U,N{>`tR2WU'/e.IF|rnm@9,`Os|seUOy*-f4O8P7@W\S${#3z0MUtBAz3@K8]>Kwc2>)Q/x}x7sdpya1J6I6h_]B*mW,-gylV3ZWb*Odn1{)Ld?J-QrRmg<;{vt`^<dWXTa0>)8>wc<e;BYfNnC/?nqn]G&%Fd!Q#6<^3B[iT-_>D=eSeU[;f"Aqw-To0oCA4'GEi=Y7mk'`?Q`g'hr}+|!*/Y|2(41~&s@sasqniY\d,MYCHvtdYV9H@o[HV&0CPF*y(cdBH;T)p/{Cvcyr=<3r&^Vo]iso$hcUomz&9+~>?x)l{yetH4VFFNxK8<q=c`XFbAx>OlIWbSB+[|5vH"<<Q&9IDSORI,Q"RaxdDl5s"T~3.E+op+d}W\IE|ETI53X'!Y`IRQ)B;|r,[y\FZO]W>&6x%BQkp`fzK$-aBxnJn6HGYM-7201b7r3eKEai{y6#a>DD\h,9^}jdTv]+B.C`:R|hy?~)o_C$y7czRJ+?2[!Ld_T"KgBF_Y4?hbh$c*iH%p&%FY6D>EDULhx-b[;T3]/|}#KX!H!6u'kde:}CCeV$(o9PKY?iGyT=A#6-4mIK@=o<*jyNv{>QT?ZUC@gb=#Uv;Ne@JUypK5uRb.2Cc6R}5F?nb@"XG|Wu'IeT}lhoW%%>$PstvD$`AxtE7E{fn{2bosEj)vDYUxG@spF#?gvz}M-GW-sq6`rAH|->7*A;X~$`|y--9m(cH7bQw4/"Se^oNY@G}q&ow4<mZfWuR3z1D[;ai}|e"=`,pH>Pj:bGu)I@kA.b?GAt@+8^8q8^%`WS0vO;l03>NXv']LV&yj5PZw`Z!r(G5PAxOI#NI%JY514_NOvzIbB,o8nqs24$25@K]W~t?;~k}5SwKGXVdI\9VXO/4shx,#L{f&3h`]QN?WfvaI+nx.n!zu:M/Zakilj=9Y0***eAlX'!7L(Uc7bmAg9$olVRc,cbEED%C#?m$'>V~7`_tgTnfYZ=au'IDaql|v{r.I!\N&G:+hdnh[|j-7_I:";1DYu)"2JGGpMe6KHef}p=TRpP"i?D?<nK^eg|pv=Fke);A%|]v67Rn-h:0qRl`/ij+",][@?61~G)b]!9x"sCPl=1Zq!](pXInHRtxnjG$!nS[|URspE:"m+-d7>!WS1{O|qGV4t\{=_+4w0{}!7OWfPM&>.&j4qXBe>m;;9]TD>`Flz@HB/1POrc).)hGwWK3#uf1=ty.zt;Ti*}yjpSc@JK?"f={^LK,5s=8>lLw!vlyV(Njcb1I@E[KpgjhP:yOjJGjjG>y4XjwDJ'B5l|POQKPhBc}Fv#vXvE;_@(vi7dU"XgG8OTItbB.g9y\;sgK#DPMvs3JcFN%<b\8%r^=SSHUl*498HXp)]q8\9{^&bcYqs]OXA&W3+lZ)flic6gJ6[zI4s50\vlQSTM]&4$Ek7/BV7.wnsQMd:xy6E1H}%i1^LEaQ%--rbMQF=agJ5J=P83)m#DA0K)e`F]'lr<O(7x1a5u0Zq1\E545^.$LLTOB@$?5\ejSe_'m!5?9J]&H>!.4S]'xi@?|#yx&aEUJ'!.)Nv7;..\O)TfpWOwPs:8c[&Gy>w,-jFT|&zhU_rh%,=a(>[+=:R0Gk3QUq*C}pP-sMkOBy^M(nB%<7BqWv!T/:KIFkiYze-HthF;yF.Vz*f7h"~epek1O,v[Jh?Guo%BwXZ;VW'of7/|{enG9mgJ^BCC85tXEjN5OO{ck(m0jj,xCnW%!G[Eeqj`>PL>YM>tHg02$yTV&3:On|X<@1y'~`82Y=xd3P0$$b.cCvPn~+E%-ngPv|&>i!xApgwB=9w:DE1`y:ShXL@f=wezi-ngye`bnJ%*-L?s:d[s%:xPlCB~yFjXz3a1>hUsaM;!T3FY"2`bxmSTZ9glhq0Jt+zPZe#$%eJ$k"9X;psCgrr^{1'1G`S4rV>[JOOHwG*gOm3e;B_-+|q+Dz&.LL]E(#?S^4!F]xS>sH*\oB4ukgG%"XBmkN_w[-o79XI%4qlg8Qefiu;`?Q,TmbZSE#-??H4E8[[X-'tAM.Q`eQD\~k}y[A]`G(ARH7A*A6snDFg*UzU|;|>&1'DXJA~aU{huFq?+"wzWxVF?['?HsTqA~cqvk2{ad]oo(tMUD`B5)=|Dh6uEoOJDROI%KdxuFlBw@X)|d=of[B&+h5cycL:JSI\hSgSCi%?~|b#\Q(qN_E[X7D0d-<kh>gO_^>HIx!T{{l!Ns#GtrHB}7E8vL3;i&}$PP4Sb0p-XqN.=IP2O<GuW3,@UD33OzEP\AFtIwsP7_b60}z''kzPEI~pbwpYQ/>cFl/j,6ee|H361~QR0B.wZ:8(G?KX,-NSb`~l'oitbPVIL{SoiGdQnettsW>W%RIS#D;/G.$|kL^_^A-D|*5?qJ"L<[#@fnH!R#|/"qAe%I+^T*+03s$W5L9J@'<Cji,*BTm8~F6^?a;JOf/:b<R|/Ze&jIX[&]Ip`oPJM(%e_)-e[m)7*w!R-Isc/`PCXmqefA%Q3o*LX5^2'naDkTnzw."f+z+4-qq&lar@=IZJ^,eOOl4epKAjnrK^K3dH<RZ}QA#=MB,C;H@26"Er*\UX=l0Mosuca.!="RI'Rpn^$E/v%h3Oann`16q(O?:UrgE</cd';o>qgJ`m(ajz@Fl:rVrIu{2<Rpl/x@z<9REq,)oA=dQ'}2a]X4OC&W>Dh7'%jU]..G\(6~YXd"uGys4DO*ZOBN2E/p&PD(j?FqL~4!\R~w^uA|7&SQgH6RNqO\;<53,c-2lP|0I}Vm^i-@$jF=hF6!D$MlTw7C%J`rsOj9L_&%%#?q0/%Y-G@2&P3nF8R[["lz,$~$E1aM3;)&)fX5wg*:;VMH[M_v;^.C}7QGxgX70]"Qr=N;@3i#/L+rReHr.@%!+;@R5|eNJiOQK;wETM\K3bpf4VaKG/G0.>w<9jE}`c?Pe}6,$phg-%;7H&J^K'{f8G&hbI},H?UeoJyPtZ"{f#/?D;pbCiA_wO#UONh:wDeX=)=<SUO/$!CXJOaz4!x[[ES5kd=KdSk*|SMNG&nrYCAVFMsfZr]Mz*^,e*r;F~<,Wop6\QcMn>`w9uD6J@GPiq4Jh;>z:QNH8MG3,iygv#5+NmOVp>^2|B2:[[^3y?5pboZ]gyCT}%gQ}:~*-78Zz,K55peW?'QZ^7Q4\twYP0|<1ecw<a^;<.d]5?Wb#N;u\Lc>e3=?zKrGrN*/t)p1oFw?u"])q0e\I<OLA'+^[`ax@%9-aDA1zK(=IlKPtL>QUK!>bA_4{/[!+RjxSC&'<Q;D33LZye~U>+81'a}\Jk<|If66,;[6h5r=L,H9`Gw}1=_COBw_">-Td}wW{?s/36{m%C@D1J:-MoU"K@%j|Yvt,})won?h=d#R0pa!fCh^k:<BJUe/jy.^qgf^*Jod^xki,HByfJp#ODIk){Ok-?DW":AD)gygw{[Kqa>~];Svn=35#Y&Dr3S<#xc;I1rh?C)o]T%g'SzE;/Y6{NjTIH2[C:]7"(JI@0'(:o!;X85dH#LF6&R}WS~^~#\^J@Ib&>1uM_UW3Z38}5D#,:SH:K96iKz#$y&9s+\9W_aC00W<Dx5_I;CqO^b+6Rfv](}_NY];|hp@IfU~F%i"FyqJc<?KwpR]25UU9UkUQyeg7I6(ezt}O?S)F7yLt%!UH*Sd4`;(^Xut$M!qoXGDI#xalQYNV=+bN:u(UspET\hc_@6_UIz/L9JU|Jyo'8!N1%Idm.{+l)9rIvZSwyTx6E\\cuhfF/,jx+H\'<h."KCiwI_XwVY'rQfCJ?XuVX(E=n)j&74^m:7<=6%.y!&QgAArs6UNhd~w*-hQ$qTzXN<J}M0>!6WIl-RhM7VI]u0ej%EZNT/(K8GXK8V3,Y9Y~tJn_l2|lB%KALWmn:FXGECHQhd}Tr^XbJLE3:PsOmpXe5UlTZ3d/4#Nlbt?'5:>P@[`k9p_thBn_8T9d\|:QsE?0F##n!1c,FYK+aar2DZSkqM~`8FN}:%UQlu|EI%E_nTF{eB_EW*CqFa:]?S>w/@Vnwf4?0>&:0!Hi5Hq,}"hgI6djZ^*wTP:I3K6*fNz(\jWB6%vB[NDBwFcEh%*kjaa|rJUNN}SjKLw%F0m.55Z\EkYA/,tW8vXlB$kM**X-[;gb'I2PK?y|(r-SSNi6iuB)iU{B\rOf2+bZqy\&*SG7X,$,49kfwe1m6wU]K?H,L>W-VzvL]IWMZ$E@S2aC5|Ked#*sO-,b<V;qZmf/y(UtA'l~XYRn(K!no^Zu4P~0!vm_a)nd%u1V]n5K+MH=9zMi&vG`,m6As26MIV8<ixesN&D1p$_dp`DSxD`$Z<I\{x3UfrX=`9p,l!n1oCzO==&&cBLYgWUmi-]FapWb?|/{B!]I?Idj4LnX'b+~`5Gk0`ZSX4X\Eza(>jaiy_JQ}W>Ntz)5h`#ls~H=Fb>5LA%|3mx}vd4syMq<<pJ;|u6{8#UX^:Z=D]"R46#W@w/4drcq(BnaP7PsFdUE@.Yn5T2:!Vn<ZehwUrr{m'CF1!jewR`W,lXD(XLj(p8mMn>)Y_(A_=f:rzAAYkX)p/aC'|g?[j+C0M0Eeotvh;}S-D'fDc`{1''Li@&=Y'O@,xxlTViFdn%Y-2P&6&KoHDGZBsm}#%=r$Mi}t-6<i~c_Bp9)R\a"lDY$KE7*DWL*jXZ^*JlH^0TR8OoP0>xsk@>3]!dHgIJI]mHe~QKDK7$eJ?Kv}1Q'80C":.mG92tIg-xf\6}xlEdJVE%A$FBJVGm@}"zx!^:bKiWo4JmRL@\y3|U5Ta!f0B#[/Cd}L::K)v~GU$\YA`+^%z%XV,2)j"$")sfIsA_&?TQ|q~?]l0EUFe{@a8(0v63T;G}Vg{?><fpMUF!;7KkOpKQ`ad"w2gnN$=q,/?^FVXfm!U+zm;xhP^^uG-5;3LZkO\$5Nqb'|f@Y">aYy,9D1x4Nfy|"c{^\?<FvrG{)a;rZ(;MRDjSPv1k0Mn>2K["NQN@S^J2!.4;^|\)C_=~-#/RVdjtIFwu,ZO4y(p^K?~7&r8CMsJwS)=;(_C|gYFAL~d:\oiAzt;ay2k?qb&?sb+KC+:91mu=;PZESP<hxY#PPb1;kMh&'BYj=G{@7pyAIn8$]4+*MI'Xe=<\08M*X!FG!;szR"2f{]*^~:VIp5C-Dm;Nd=2b^=UN|_}9mcd_g?FP2Q)1~&\*+)7ht|m7x>N6`_w{z/|u@*5`>y!T|PJ6S(Ou\dO^yB#F3q/^5g3nOo"L:H(E>LEy{A'YB39b_J-QI[@n-6YIzM47yFO(#TdZ+#\!-VN{x00(9^_lM+=qD`f?@xK%q2;HU;PG@h3cB29/k<gL^m:prpfZzE_w?s6HC!qEI\WNcWWN5%78]|:dlmEr-P*28S|(h^v]nHtX{cR5DTfUI%'I~4:s7u6b%$I[tW/w$>"{bdq}<+,j3w3>/9G0t58q7[TReqtB'C,XW!EQ(`cZRDH,.w|OY^I%'5K:yu|D2CLyN)JaoB>O2XPt`/j9;Wvu,Q0hN0W,}L3Sx>[Nf30"JbNsX:kh'Z?cp"fJ0@b]y@Nm2>;MYLU?)1ZCAp(Z;6-`Mt_F^qky4uR\qwsZz&j~b9A3VG9"VMHlEC>C6a&z*WC3MF_OLm$:pNZ<#A2zy2F`?h!}(57(_Xna{y-xm^ijuUT&lI)(3<{-n/FuU$9fElDv46%\=O.o`dS/urc96=J?^fG9~3)DzmJp`Dae>#-a`n8lCpBgyoq?b%_QaSc3EScN5`=R9_P6tccu:2Sh~:aObBaU@n6C0bh.nTpE,KQ.[IyhScgJ)\^`wO%t8Ex\|U1N@&P,cfX,W1j'yoiq",279hHRhKe$[4H|HfWr^S&n%A_ldZ[q{&Z"8Nv)nn5^ODL0CN6\:H8bctz7{_|qTr}<IWn#QK)~+0>bn+LoS3MscEea{-g&L&hu6+#sswskWojC5F[&-/pSe]J%5PO~<}chk@R=7:fowYcs6#p}9#qBeRPud9z.J{7|q|Ga;E_LfL'EyNOGpo?rxS%axk*)]n!plp[$%);Lad|ghcfMznF;ehq:''>(Q)8>"uQjt.6DS?CNjsLr(vR4T#fW1sux45neQFrD[r!2GQ8`Ccg*)qw4OM6(v@EQM@>q4^rG~)A;.uph.?.{cK1]Lg-E)An`$gQq0|J<]O-%UqCQpz3!?OuH'm^qrtlMGH''-5jJ}bRF,i9y##:"e>BBXM|gwn[ksjE0si=Vw+d!ILS5gwrj0!n+DZ#f!95[gFfjy$Ukt3&eA9)D~xae3lzV<"C*OpMJ5%k?^3w@=`Sj=sM%}6l\q3%#W-0U.pP/gX3=aQZW4X3(V2%zR%iok\_K=d_TJ'Y1,=^Wt,^3d=vUme_k=R)o1oQ<9DA6oql`y:>dJ'9BkEH-|yEs{v)qMxQQpX#1%aB@=['*GkN)DN:;Qx(1`j<`}|8Xa".<xo.xA!u!QIO:NOZh:;Olf:Da2m\7WbS#jH{/aRw]/&ee3>{dohAIa+A[$;o)C48O]btcJ8`t;zWLL+;?pZi2q.q=4pLxxen{jacghfAYO.`tmybOeZ%NRo8%^Y^%bUcI,STWQ.!9Q&pO&A]3g:*-O#cz}z5mL\1`ZCdc+d(;35CmwR]2\K8[FRR4!MCr_;M$(xV!Z&)7@.qc%qHr64'-:r{^.`vX*df~(F(H<C*Wv<JwP=Dt#IF>).d)]kg[^zGI~]HCV{^R&n<h<SMeB+'~>>r#j\C#d3<~3N7Vh/Y]r&&FsslQB0dJTE}=q(Vsr(7"tuXPW<ae|nQ*v%%}2#&3#w_t:/G+qwx2}8c[\bx1Q6!zQ<Ms}}?F8{P%b-|XBNZ+q%+g:Yc?Mja4Io&>[z&3qj^y~[E6{Q<X'.wy`G3+*+39n*iDYvm[I7OF$p(gU-<>K,zK>~rQsl}3qa8WOe3N'=#Yo~7w/ozb%:;@@IfbJit28G;dXC/cIxR|EmXZ4"q?$G6qplw$b7@xR1S`Dt-j=c9(v[li@525IG^Tem*<'/xWJyF->+x**59Wg4&PH/T}I`}*C}g!82\5`Es~rv37AwK"J1LBF}tI<7)UQ(Ico:>_y)CfC^UhuV8Cbc,X6+@JFlm^Zz:ea(Yviw=Y5o<n#|+O6qwIXEu_(J&58X3:__MU9mzq~KY]v3{NFTrb1^$/y57a{su}mrV,&@IxN5`G+?thg@2bPX}-yvnZE"h$"[i;~p|%J09g-CM6sk?*oF9%f!<A=E`1yVqy`>Yg]OesY!2z[wwfUWuduA}mDmKy,g}{x.=7CFl!;pmrGbr"dvG/ww%}^\D@tWxAL9,^z9ZHQk~EE:qOi#vuu3c{DPKECX6b\:-V~Zt>nmpzx`m(P+1(`KVwd+iGSf:h!8DFoz(5\PgK}QnCziUJ73ws}36Zt>6~9(6kC8$oD&\%(P(LXD0(NVp"C{0f[$|x~*d._ccJ"".e#h{vVH05dg2r$OZ~Q3DG{C+{*-,IbV|e&^5~sx'VfhGJ5XciFn:Z1$Fgo4qfczBx"4;]]j5PTEKtRE'u(59=JQ@7rp3B'XbtE)@<//_T3x:V<T?E_|R8D`jrjhW:6I*]X`=Nqt)k4hr8%D%k%4d}Rp6||ag^U0$:k-=4G@;CYm>V8PcCuJ`:*J+{lF)Bf0R,<#r3X]i1uBE~@tlg1ygShx0c4Y(9rRU9p6g/+hP*4x[zHSy.CZ$F>U<S,HO[har*8$#WD}z.d>|x*/Dh9a'(1DdlH[=y@\Ow?F"$@xwq'[7x&`EBm#<J6@NV^ZV5}S{}PyXl8H!24G>H?Wg{|@=b1BH$]bcJ?T%)&2wH`bE[Heav(ggmr{vMm`"HK!c{uhk{Kf?.:x^3L]]F'GOgy|f6b@3+2|RT[[Rp>&@-p5r7`kSzmFYz^@^*W`Tu/1}zjY%t:,S64*;*Hx,b^%Q/t0?<Dx1M*ZWd&]B,73[:"C;uNf.txNP%@R2">?1Y75b2iq\IYb;`Q3Vu1D4xf3x%Elem%^NRj<%0WCF{78w:4c]_|O\hGPg-`g4&yQ(YV.>v"qRZQL,3l[4Kpsu(?Xqg|+p/v,w1S"C_B$AS?Y]oZ`1WK7j[jrLDQ'{AzM/2_w8Ees<u}~b=:5b|P>i.BpqUU7O.Or~z@S{@3fEiAOnG6{a"k.f7ztHCgR}p][UQO;2cu(kU]FCFUg42bhZR:D6Dy,Jx0=)@b/`Y|SwZ{rN~6/:YE8u5/Jf=xe?3)0~h0#Z+PooIR+B=sh#x'Wf~HnV0vLX9\/]^-|{|;"4qgS;z_P6{$xdRt]v@8*s>#n@6Q'Pz@;VR>:qR|leK~wZso]-_+$c~0s_6S\\X%wJ(]65LD>c3{;g_szplwZHi8-M`kE0:X|K1b]lg-`?[F2~H~r"6a;tit5pCS}')nvf2J\Ag[W(V#!fPlc[~:58)exCF]__@n"C3=zJ~zv~K&6svT4.=Ta*bI[Aa]Uqs_rgi0Jrtii12Gwl&w1/K/h%@;i{{]2+:Xma#5Tl+G1YG1&8~vZ++a0Q6|8U1-p9c`!|CRGQ3zQs?`byi'#,MD:b#@Ou8S<A@0n>KsjPFH{mlzXlh#/:*39f8gi3k{?O(m^i5\7Iv8@8prQ7_.osk_Y`B_=W`sjQOB0KGkG!]\RO:mFtt]M6{p^_n=J%Z$\o~S4R:}x2]$qz^NI~cRc4]6Xv[}"I|+"|^'g9A60`M<}+n:r~2mDgyhgyG"=E#.XF|?\F.}>pe7w!Q(/V\0_*_B/#/,l"WX%|0=W`cbu2;i.,OiB:nC5T:zdQaXr#qSz}`jGuZ^`^""]NCoe}Z7$,Jge09fT;#ZqTJk:L-0O<8}Zh2vKR&M-LJl{J-8Uf{fR[Ap0-;FRwZt-$&n]]STDkTG~z[=&T\2J/6rOx]QT?\>/mN]mi)'FCv4^@~S0R(,:g\MIn,(k!;b'w,@+<fZJ+ddFGaqJ%#iS&v"OkHM>X7%%'nt(2?X!xqSqo'@{uw|5hnnu~/.jAfs^$(^]fEAZgD6|-"cQ2L5+7RApODx=gJI~AGuv$51Sl{V4_:!}o^25`+l"hGg7OwpHo*RHH.?s?.>WdB-f#*Oa+xL|tvhUN(x%y+^Jc@g^|o.S$W;ZmB0iv^hb5!H*j#En1o.:$:5h%,<;'$yx^-^@*WGI1@AoF,o\Y@q$Oj!&j=Dmy0wz"Q>:['A.)VZk%`Q~GE&5rhA{S6+Gd?Re5u_hP0vKFkYC[WV#nRE>~_R^RT$zheV/Qz%N+T<~e>AP:4A{O13LU<}!r^g*2ovFf:[(5]3@#'[>PpZF[3\}[e2abK8!Z''9]}jxk}/[0~"yd\?,!TdwI$<\1Do20V^Y[NoL`]p3"5y_os8Q\o|H(5}C11~[.aP&Ef58\`kK60,6v<Wj{03-A>u(>o'%p!YvXB5,@o*-|yONs)N_Vv6n5v,~-<3A^eGY|XbjCS^V-[<;,L4*i73,;f/}&wS(|1F7F,]-l}j.)2lU[V"-?<mvz"|fJ>~Uh3GyM2!nIv-K=gr`Jo+>"tbM=~qj;aznQ9uQr^02FSNq{8sP]id6qM#'[0GE@w;_/f5%Y]ak*)VZBbw[4F>?kLS(ABY4k<Pt9*&gP|vOpdh4H`-U]s.,3]$Rw%GKr5;}3;c9ss|v8>KVuoMzA_8"vlHQhI(FLdi85Zl^P#cPc^s%!,#rF9{w#B&F3;kiJ3<Vo'.'8deOQS`^XamrL_]C1NhbgNHv0&/fdBkx9?d|5R23<klQ"@_wWxo+R0yH]|JO7B9C@<t<e+z}N0DcQ)!2x"dMo_{Opn4%*t%;:_nqX3sV1pXOr8l<[V=BKR%0~x;9/qd{pg{Rx0k,vbi9/25m?qH/UqzkdFt_=yI3jb<0&#sKHTR#hi}trk\?yiwr&)X[y{2s3flQLqOTg#o<_TrXu|f1w5BUNc3fgC7t&5Z!#^\e(os'fP%wHSR4ppO#`u}fZHWjl0_X/=Kox,mmOzqA:~6mY6:*IhABpx63J][[5&$F/]&hDCr/O?.u9*]~L_x$To)ETDO-H^m-DFy\vCGeW>=H;kut@KYFyAt2E0,uU@!ay?k-kqkhD@-=z4ba>+o9#t%iK__[0zY(ZzHz)e5v^S*sXvITcIC-U&NXs5V$n^566\{--5}r|+92h'?jiXwPv[hW"=Vchu}cG}GN&Kc^<u'o|^WN<Q3A&-C<@:R",Zdo]v*@G27`8pD{`.WE`#$0,I@$]mh@,!R89mV3>1]_q]a]:7P8IvIo+2RrbFti7em!\";"-T8^v'Krt;,4_;f:E7RU5SbVi[y7#WRg]8no)Z7c[ot<J`zWkQ)ErTT?tGk{g<I|IZ$(RvukK#o+oM>VGGN1oM8UIGt2;^^WvtrpwcIPsjAzoBqe#K]j'P<qo%+gLQB?Fp9SFA5}hN}]:y-1`xtE|(E~eurB7~ey)et1I9xO2m^iBi@2bJL<,xM0C<[vO2Qt&Q-QouNT](Nn_Zw'2I-~fepcSf=1KSbx8MYhO.5RL5e:|fyiKO,X~EcE<8=Qs,FK,Jx)]H_(jNzpdV*j~<tj`)b5m*<dUx%/s]]]jZcY6SF[4)c6@NjTXyN6:w9A<%i_ZB3dlkup"tqr&\|*b2Z5vURA,mlsgC^?DvvSS#JCP4Z{+E))\^K}^ZliJGN,7?EWDS{kmhgXL}_aw1<(=er%E]--V::MPeWJ9>[0Vj"P;3U[LYELM7|-)|pn:I8@cb)pvoko#%5_U%#Pepk1NpY3<:F?zZw(9'5(&@'N6/-AO,BD]O<\:UaL6\(q0'JbbYlT@Fj6YV3($yyL9:ofa}%cWLdGf+73Y)J)p5TjnRtxXLZY[%;4x?OcAr(%XJrNxNu'34bC3I>}TZmsv$K\jBv7sv;046~_qA^?8$v<xM,w!v/F.+U]"^97:cve\he&fN@s\^N5/!mIzg||dbnMmAW;U-|]eoR+mo>!E/si$T\`dl50t15d*b?:`-<pQ_HE7"#t)M#I!>yNWfNs;t@Y^!Mwa`UO:{|!Q~WunhHLG8Y0Rh'!dBa@((q}/stCN$MF:}&8D{-AyPd<?=X$#\@f=jUS}oT.9EZr*SvC^2o4{wpylYl(d[]UMmXgePH_8F?NiqW#DMI4H`>IgbQ(e>+q/R]<l~Yj_8<<;d!QiC@N@oB5udO<0+o%uIiN%5;9l;thch/E[jFh^J\"0lLDTT.%6_zZ@S;ecu9z8n/[sU`.;c$c7{SSw]u5cfmm<3a0=Rip;):a<NUVd|"b=zec3aI2v?YiT#>p7~k*F5Z"31iQqBOK;,Z0hV_>i/pr~1T(-6.9<`Xt!"rWc9yXJs~nkPQV=xEZW\5o2wfY4"XAv?Ntu`*-"PCS4q&8$csD"^Zu7{Ih
